cmd/prometheus/main.go

```diff
@@ -222,11 +223,13 @@ func Main() int {
 	// to be canceled and ensures a quick shutdown of the rule manager.
 	defer cancelCtx()
 
-	go webHandler.Run()
-
 	// Wait for reload or termination signals.
 	close(hupReady) // Unblock SIGHUP handler.
 
+	//Set web server to ready
```

Comment:
Please keep the comment style of the project: always full sentences, end on a full stop, empty space at the beginning.